Update ##Prerequisites

v-carpenterd opened the pull request:
## Prerequisites: Added note stating that the Export feature is only available in Consumption Logic App

v-carpenterd committed:
Update ##Prerequisites

## Prerequisites: Added note stating that the Export feature is only available in Consumption Logic App

ecfan commented on articles/logic-apps/call-from-power-automate-power-apps.md:
```diff
@@ -32,7 +32,7 @@ If you want to migrate your flow from Power Automate or Power to Logic Apps inst
 * A Power Automate or Power Apps license.
 
-* A logic app with a request trigger to export.
+* A logic app with a request trigger to export. Note: the Export feature is only available in Consumption Logic App
```
```suggestion
* A Consumption logic app workflow with a request trigger to export.

  > [!NOTE]
  >
  > The Export capability is available only for Consumption logic app workflows in multi-tenant Azure Logic Apps.
```

v-carpenterd committed:
Update articles/logic-apps/call-from-power-automate-power-apps.md

Added note stating that the Export feature is only available in Consumption Logic App - Edited by reviewer

Co-authored-by: Esther Fan <ecfan@users.noreply.github.com>

ecfan reviewed (approved):
Thanks!
